supported_interpreters.md

```diff
@@ -311,10 +302,10 @@ Please check the below table before you download Zeppelin package.
       <h6><a href="https://ignite.apache.org/" target="_blank">Ignite</a></h6>
     </td>
     <td>1.9.0</td>
+    <td>1.9.0</td>
```

Comment:
Is this meant to be twice?

Reply:
Thank you for review, I forgot to attach screenshot for this page.
![screencapture-localhost-4000-supported_interpreters-html-1505443106320](https://user-images.githubusercontent.com/8503346/30464106-83bbb942-9a0a-11e7-8212-aa6d4b269562.png)
Zeppelin 0.7.1 - 0.7.3 is built with ignite 1.9.0, so yes it is intended change :)